### ISSUE TITLE
🐛[BUG] -  An issue with the get_rotation function of the GPS module

### ISSUE THREAD

**Issue by OWmess** [purduesigbots/pros#625]
The code I wrote is as follows：
```cpp
imuRaw = degToRad(gps.get_rotation());
        pros::lcd::print(3, "imuRaw:%f\n",imuRaw);
        pros::lcd::print(4, "gpsRaw:%lf\n",gps.get_rotation());
        pros::lcd::print(5, "gpsHeading:%lf\n",gps.get_heading());
        pros::lcd::print(6, "errno:%d\n",errno);
        pros::lcd::print(7,"yaw:%lf\n",gps.get_status().yaw);
```
Then I found that the gpsRaw（gps.get_rotation()） data printed on the screen is always 0.000,but the other data is normal.

Looking forward to a reply.

**Comment by OWmess** [comment]
The version of PROS is 3.8.0

**Comment by noam987** [comment]
Closed In #630
